gnuradio-runtime/lib/controlport/rpcserver_aggregator.cc

```diff
@@ -10,6 +10,8 @@
 
 #include <gnuradio/rpcserver_aggregator.h>
 #include <gnuradio/rpcserver_booter_base.h>
+
+#include <algorithm>
```

Comment:
Oops, this should be in a separate commit.
Also, why isn't this already in the code? is nobody building rpcserver_aggregator.cc ?

Reply:
*crickets*